client-app/shared/account/composables/useUser.ts

```diff
@@ -112,6 +103,22 @@ export default () => {
     }
   }
 
+  async function signMeOut(): Promise<void> {
+    try {
+      loading.value = true;
+      const url = "/storefrontapi/account/logout";
+
+      await fetch(url, { method: "GET" });
```

Comment:
Do we need to mix plain fetch and innerFetch usage?

Reply:
InnerFetch does not cover the case without result and without body